cli/deployment/config.go

```diff
@@ -471,6 +471,26 @@ func newConfig() *codersdk.DeploymentConfig {
 				Default: false,
 			},
 		},
+		Logging: &codersdk.LoggingConfig{
+			Human: &codersdk.DeploymentConfigField[string]{
+				Name:    "Human Log Location",
+				Usage:   "Output human-readable logs to a given file.",
+				Flag:    "human-log",
+				Default: "/dev/stderr",
+			},
+			JSON: &codersdk.DeploymentConfigField[string]{
+				Name:    "JSON Log Location",
+				Usage:   "Output JSON logs to a given file.",
+				Flag:    "json-log",
+				Default: "",
+			},
+			Stackdriver: &codersdk.DeploymentConfigField[string]{
+				Name:    "Stackdriver Log Location",
+				Usage:   "Output Stackdriver compatible logs to a given file.",
+				Flag:    "stackdriver-log",
+				Default: "",
+			},
```

Comment:
For sorting purposes, what do you think about inverting the order of these names?

```
log-human
log-json
log-stackdriver
```